.travis.yml

```
@@ -55,7 +55,7 @@ jobs:
         # Install and run `yamllint`
         - pip install --user yamllint
         # yamllint disable-line rule:line-length
-        - yamllint -s . .yamllint pillar.example test/integration/repositories/pillars.sls
+        - yamllint -s . .yamllint pillar.example test/salt/pillar/pillars.sls
```

```suggestion
        - yamllint -s . .yamllint pillar.example test/salt/pillar/pillars.sls test/salt/pillar/windows.sls
```

This would be needed in normal circumstances.  However, it won't work at the current time due to the presence of Jinja tags in that file.  Will it ever be used by non-Windows OSes?

---

It can be done without Jinja, with a little more effort.